inference/inference_benchmark/cc/Paddle/bin/run_clas_mkl_benchmark.sh

```diff
@@ -3,20 +3,21 @@ export RED='\033[0;31m' # red color
 export NC='\033[0m' # no color
 export YELLOW='\033[33m' # yellow color
 
-test_cpu(){
+function test_cpu(){
     exe_bin=$1 # ./build/clas_benchmark
     model_name=$2
     model_path=$3
     params_path=$4
-
+    declare -a name=$5[@]
```

Comment:
could you rename this variable, since 'name' is ambiguous.

Reply:
Done